build/azure-pipelines.yml

```diff
@@ -261,6 +253,37 @@ extends:
                 SymbolServerType: TeamServices
                 SymbolsProduct: DevHomeGitHubExtension
 
+            # Only run APIScan for non-arm release to avoid duplicate results unless it is disabled.
+            # Copy surrogate file to binary directory to use relative paths which are not architecture dependent.
+            - task: CopyFiles@2
+              condition: and(ne('${{ platform }}', 'arm64'), eq('${{ configuration }}', 'release'), ne(${{ parameters.APIScanDisabled }}, true))
+              inputs:
+                 SourceFolder: '$(Build.SourcesDirectory)\build'
+                 Contents: 'surrogate.xml'
+                 TargetFolder: '$(Build.ArtifactStagingDirectory)\rawBinaries'
+
+            - task: APIScan@2
+              displayName: Run APIScan
+              condition: and(ne('${{ platform }}', 'arm64'), eq('${{ configuration }}', 'release'), ne(${{ parameters.APIScanDisabled }}, true))
+              inputs:
+                softwareFolder: '$(appxPackageDir)\${{ configuration }}'
+                softwareName: 'Dev Home GitHub Extension'
+                softwareVersionNum: '1.0'

```

Comment:
Would it make sense for softwareVersionNum to be set to softwareBuildNum?

Reply:
It would, but APIScan requires us to register the version number with them.... so unless we want to register the version manually every time we update our version, we just keep it at 1 :)